lib/Client.js

```diff
@@ -1420,6 +1420,30 @@ class Client extends EventEmitter {
         }).then((guild) => new Guild(guild, this));
     }
 
+    /**
+    * Edit guild channels' positions. Note that channel position numbers are lowest on top and highest at the bottom.
+    * @arg {String} guildID The ID of the guild
+    * @arg {Array<Object>} guildChannelPositions An array of [GuildChannelPosition] (https://discord.com/developers/docs/resources/guild#modify-guild-channel-positions)
+    * @arg {String} GuildChannelPosition[].id The ID of the channel
+    * @arg {Number} GuildChannelPosition[].position The new position of the channel
+    * @arg {Boolean} [GuildChannelPosition[].lockPermissions] Whether to sync the permissions with the new parent if moving to a new category
+    * @arg {String} [GuildChannelPosition[].parentID] The new parent ID (category channel) for the channel that is moved (only one channel can change category per request)
+    * @returns {Promise}
+    */
+    editGuildChannelPositions(guildID, guildChannelPositions) {
+        const guild = this.guilds.get(guildID);
+        if(!guild) {
+            return Promise.reject(new Error(`Guild ${guildID} not found`));
+        }
+        guildChannelPositions.forEach((channelPosition) => {
+            const channel = guild.channels.get(channelPosition.id);
+            if(!channel) {
+                return Promise.reject(new Error(`Channel ${channelPosition.id} not found`));
+            }
+        });
+        return this.requestHandler.request("PATCH", Endpoints.GUILD_CHANNELS(guildID), true, guildChannelPositions);
```

Comment:
The `guildChannelPositions` should be formatted correctly for Discord. E.g. `lockPermissions` > `lock_permissions`. Also, the name of the parameter is quite long right now, `positions` would work better in my opinion.

Reply:
the length of the parameter name doesn't matter, this isn't part of an object or something you have to actually specify

Reply:
> the length of the parameter name doesn't matter, this isn't part of an object or something you have to actually specify

Still, looks cleaner ;)